From f316b794265f28e89821dfab0772caac1bb6b056 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 18 Aug 2020 10:31:47 +0000
Subject: [PATCH] src/hb-gobject-enums.cc.tmpl: write out only the filename,
 not the full path

This is beneficial for reproducible builds, as build paths can vary
between builds.

Upstream-Status: Backport [https://github.com/harfbuzz/harfbuzz/commit/e876886d1e877ad90b6f02badefd6ebee1bc0b09]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 src/hb-gobject-enums.cc.tmpl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hb-gobject-enums.cc.tmpl b/src/hb-gobject-enums.cc.tmpl
index 2ffd1c9..87a11dd 100644
--- a/src/hb-gobject-enums.cc.tmpl
+++ b/src/hb-gobject-enums.cc.tmpl
@@ -43,7 +43,7 @@
 /*** END file-header ***/
 
 /*** BEGIN file-production ***/
-/* enumerations from "@filename@" */
+/* enumerations from "@basename@" */
 /*** END file-production ***/
 
 /*** BEGIN file-tail ***/
